spec/features/status_update_spec.rb

```diff
@@ -0,0 +1,25 @@
+RSpec.describe "Status updates", type: :request do
+  before do
+    login_with(%w[internal_app status_updates])
+    subscriber_list_id = create_subscriber_list
+    subscribe_to_subscriber_list(subscriber_list_id)
+    create_content_change
+    @email_data = expect_an_email_was_sent
```

Comment:
Minor this reads a bit odd - it seems unusual to be pulling data from an expectation and to have an expectation in a before block. I appreciate this is part of an inherited legacy rather than your decision though so not a blocking comment - I also can't think of an immediate way to fix it without breaking the convention of doing everything through requests in these tests 🤔

Reply:
Yes, I remember it surprised me as well! The only quick alternative that springs to mind would be to yield the data instead, similarly to [using Webmock directly](https://github.com/bblimke/webmock#setting-expectations-in-rspec-on-webmock-module). I agree it would be awkward to try and decouple the expectation from getting hold of the data, as they both involve a kind of "search" operation.